test/MUXControls.Test/Infra/TestHelpers.cs

```diff
@@ -97,74 +99,79 @@ public TestSetupHelper(string testName, string languageOverride = "", bool attem
                     ElementCache.Clear();
                     Wait.ForIdle();
 
-                    // Before we navigate to the test page, we need to make sure that we've set the language to what was requested.
-                    var languageChooser = TryFindElement.ById("LanguageChooser");
-
-                    // Sometimes TestSetupHelper is used to navigate off of a page other than the main page.  In those circumstances,
-                    // we won't have a pseudo-loc check box on the page, which is fine - we can just skip this step when that's the case,
-                    // as we'll have already gone into whatever language we wanted previously.
-                    if (languageChooser != null)
+                    foreach (string testName in testNames)
                     {
-                        ComboBox languageChooserComboBox = new ComboBox(languageChooser);
+                        Log.Comment(testName + " initializing TestSetupHelper");
```

Comment:
Not a major deal, but a lot of this logic is only really needed on the first page (language chooser, first mouse click, etc.).